clientwrapper.go

```diff
@@ -0,0 +1,43 @@
+package sdk
```

Comment:
I would like to avoid refactors as part of any feature, can we keep it easy by now? BTW I think this is also being splited on the [dictionary.go file](https://github.com/status-im/status-go-sdk/blob/master/dictionary.go)